src/auth/src/credentials.rs

```diff
@@ -26,6 +26,7 @@ pub mod anonymous;
 pub mod api_key_credentials;
 pub mod external_account;
 pub(crate) mod external_account_sources;
+pub(crate) mod idtoken; 
```

Comment:
I find the module structure of `google-cloud-auth` slightly odd...  it is deeper than I would expect 🤷  I am not sure if this needs to be nested or one lever up.

Reply:
I don't have strong opinions, I was just following the design doc